Doc/library/dis.rst

```diff
@@ -999,60 +999,44 @@ iterations of the loop.
    .. versionadded:: 3.11
 
 
-.. opcode:: POP_JUMP_FORWARD_IF_TRUE (delta)
+.. opcode:: POP_JUMP_IF_TRUE (delta)
 
    If TOS is true, increments the bytecode counter by *delta*.  TOS is popped.
 
    .. versionadded:: 3.11
```

Comment:
Should `versionadded` be removed? I think py3.10 and before had `POP_JUMP_IF_TRUE`, but it always did `CHECK_EVAL_BREAKER()` in case it was backward.

Reply:
It was an absolute jump in 3.10. I guess this could be versionchanged, to say it's relative as of 3.11. But this PR is not going to be back ported, so it would need to be a different PR that can be back ported to 3.11.